pandas/io/common.py

```diff
@@ -397,16 +403,22 @@ def get_handle(
         # GZ Compression
         if compression == "gzip":
             if is_path:
-                f = gzip.open(path_or_buf, mode)
+                f = gzip.open(
+                    path_or_buf, mode, **compression_args  # type: ignore
```

Comment:
What was the mypy error this was throwing? Not always possible to avoid, but type ignore is really a last resort

Reply:
Mypy was interpreting **compression_args as the next sequential arg in the function signature, i.e., "pandas/io/common.py:409: error: Argument 2 to "GzipFile" has incompatible type "**Dict[str, str]"; expected "int"." I discussed this more in this comment: https://github.com/pandas-dev/pandas/pull/33398#discussion_r405680262 

Can you think of any other approaches? I don't like ignoring type errors either, and I'm definitely not a mypy expert so I may be missing something.

Reply:
Ah OK. I suppose this is related to https://github.com/python/mypy/issues/5382

I don't know of a general solution at the moment. Can you just comment before all the type ignores that it is related to that mypy issue number? Will make easier to clean up if ever resolved

Reply:
Good idea -- added comment.